sashimi-webapp/src/logic/renderer/diagrams.js

```diff
@@ -21,31 +21,32 @@ export default function diagramsRenderer(ele) {
   // array of promises for use by Promise.all
   const promiseArr = [];
 
+  // Eslint escapes
+  /* eslint no-loop-func: 0 */
+  /* eslint no-undef: 0 */
+
   // Draws all the sequence diagrams found
   for (let i = 0; i < seqDiagrams.length; i+=1) {
-    /* eslint no-loop-func: 0 */
     promiseArr.push(new Promise((resolve, reject) => {
       const observer = new MutationObserver((mutations) => {
         if (mutations.length !== 0) {
           observer.disconnect();
           resolve('Finished drawing!');
         }
       });
-
       let content = seqDiagrams[i].innerHTML;
-      /* eslint no-useless-escape: 0*/
-      content = content.replace(/\&gt\;/g, '>');
-      seqDiagrams[i].innerHTML = '';
-      /* eslint no-undef: 0 */
+      content = content.replace(/&gt;/g, '>');
       const diagram = Diagram.parse(content);
+      seqDiagrams[i].innerHTML = '';
```

Comment:
Using `innerHTML` may not be a good practice here.
In the future, you may consider manipulating the DOM using api like "appendChild", "removeChild" etc.

See:
- http://stackoverflow.com/questions/3955229/remove-all-child-elements-of-a-dom-node-in-javascript
-  http://stackoverflow.com/questions/7476638/if-people-recommend-i-shouldnt-use-innerhtml-what-then-should-i-instead-use

In this case, you may consider using a for loop to remove all the children inside seqDiagrams[i]